modules/gitops-release-notes-1-9-3.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="gitops-release-notes-1-9-3_{context}"]
+= Release notes for {gitops-title} 1.9.3
+
+{gitops-title} 1.9.3 is now available on {OCP} 4.12 and 4.13.
```

Comment:
```suggestion
{gitops-title} 1.9.3 is now available on {OCP} 4.12, 4.13 and 4.14.
```

Reply:
I have update the OCP support. Thank you.